scripts/react_native_pods.rb

```diff
@@ -141,7 +141,11 @@ def use_flipper!(versions = {}, configurations: ['Debug'])
   use_flipper_pods(versions, :configurations => configurations)
 end
 
-def react_native_post_install(installer, react_native_path = "../node_modules/react-native")
+def react_native_post_install(installer, mac_catalyst_enabled, react_native_path = "../node_modules/react-native")
```

Comment:
I would use a named parameter here, to have self-documenting code. What do you think?

```suggestion
def react_native_post_install(installer, react_native_path = "../node_modules/react-native", mac_catalyst_enabled: false)
```